main.go

```diff
@@ -188,12 +188,12 @@ func wrappedMain() int {
 	} else {
 		var TTY packer.TTY
 		if !inPlugin {
-			var err error
-			TTY, err = tty.Open()
+			tty, err := tty.Open()
```

Comment:
I'd rather a name other than "tty" so we don't have a namespace collision

Reply:
There, this should be better